index.bs

```diff
@@ -4648,71 +4656,154 @@ The <dfn export for=commands>browsingContext.setViewport</dfn> command modifies
    </dd>
 </dl>
 
+<div algorithm>
+To <dfn>set device pixel ratio override</dfn> given |navigable| and |device pixel ratio|:
+
+1. Let |navigable| be the [=/navigable=] whose [=navigable/active document=] is
```

Comment:
Is there a case where this isn't a no-op?

Reply:
Interestingly, I just copied and pasted what we have already to extract it in the new algorithm. And the way it's written now is the result of [refactoring to use navigable everywhere](https://github.com/w3c/webdriver-bidi/pull/729). Before, the idea was to use a navigable whose active document matches the browsing context's active document. I think we probably don't need this anymore since we use the navigable directly. So I've removed this step. Let me know if I'm missing something.

Reply:
Thanks for updating! the refactoring was semi-mechanical and we might have missed things like this.